l3kernel/l3file.dtx

```diff
@@ -1950,6 +1950,18 @@
 % \end{macro}
 % \end{macro}
 %
+% For the above to work, we have to have a non-negative value for
+% \tn{tex_showstream:D}, as \tn{tex_write:D} and
+% \tn{tex_show:D}/\tn{tex_showtokens:D} treat negative values differently.
+% This is needed for our approach to work at all, so we apply even when
+% \LaTeX{} is not in use. That doesn't work for LuaMeta\TeX{}, for the moment
+% we simply skip. (For Con\TeX{}t we really need a different approach to
+% showing messages anyway.)
+%    \begin{macrocode}
+\cs_if_exist:NT \tex_showstream:D
+  { \tex_showstream:D = \sys_if_engine_luatex:TF { 128 } { 16 } ~ }
```

Comment:
To be TRY, `\tex_showstream:D` can simply be set to be the `\c_term_iow`.

https://github.com/latex3/latex3/blob/e989a2a152d79522e3e75fa4b7ce42d94fc5bf6c/l3kernel/l3file.dtx#L1604-L1611